### PR TITLE
Generate working version of manifests

### PR DESCRIPTION
**Pull request by mhmxs**
This change recreates manifests based on the working version in operator.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,7 +44,7 @@ help: ## Display this help.
 ##@ Development
 
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=manager-role webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName="storageos-api-manager" webhook paths="./..." output:crd:artifacts:config=config/crd/bases output:webhook:artifacts:config=config/webhook/bases
```
Comment (croomes) [inline Makefile:47]: Not sure it matters here, but the operator will name the role `storageos:api-manager`.

File: config/default/kustomization.yaml [hunk 2]
```diff
@@ -1,36 +1,41 @@
...
 # field above.
-namePrefix: api-manager-
+# namePrefix: storageos-
 
 # Labels to add to all resources and selectors.
-#commonLabels:
-#  someName: someValue
+commonLabels:
+  app: storageos
+  app.kubernetes.io/component: storageos-api-manager
```
Comment (croomes) [inline config/default/kustomization.yaml:14]: Just `api-manager`?  In other places, we drop the `storageos-` prefix for the component label.

File: config/rbac/key-management-role-binding.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: storageos-key-management
```
Comment (croomes) [inline config/rbac/key-management-role-binding.yaml:4]: `storageos:key-management`

Comment (mhmxs) [reply config/rbac/key-management-role-binding.yaml:4]: Controller runtime doesn't support names with `:` that's why i changed every to `-`. The error during generation is:
```
Error: unable to parse option "rbac:roleName=storageos:api-manager": [expected comma, got ":" (at <input>:1:19) extra arguments provided: "api-manager" (at <input>:1:20)]
```

File: Makefile (hunk 1 above)
Comment (mhmxs) [reply Makefile:47]: Answer same as : https://github.com/storageos/api-manager/pull/93#discussion_r711997010

**Review by croomes — APPROVED** [review]
LGTM after pipeline passes - thanks!

**Pull request merged**